dipy/reconst/tests/test_mapmri.py

```diff
@@ -63,15 +67,38 @@ def test_shore_metrics():
     radius = 10e-3
     r_points = v * radius
     pdf_mt = multi_tensor_pdf(r_points, mevals=mevals,
-                              angles=angl, fractions= [50, 50])
-
+                              angles=angl, fractions=[50, 50])
 
     pdf_map = mapmri_EAP(r_points, radial_order, c_map, mu, R)
 
-
     nmse_pdf = np.sqrt(np.sum((pdf_mt - pdf_map) ** 2)) / (pdf_mt.sum())
     assert_almost_equal(nmse_pdf, 0.0, 2)
 
+    # test MAPMRI metrics
+    tau = 1 / (4 * np.pi ** 2)
+    angl = [(0, 0), (0, 0)]
+    S, sticks = MultiTensor(gtab, mevals, S0=100.0, angles=angl,
+                            fractions=[50, 50], snr=None)
+
+    mapm = MapmriModel(gtab, radial_order=radial_order, lambd=lambd)
+    mapfit = mapm.fit(S)
+
+    # RTOP
+    gt_rtop = 1.0 / np.sqrt((4 * np.pi * tau)**3 *
+                            mevals[0, 0] * mevals[0, 1] * mevals[0, 2])
+    rtop = mapfit.rtop()
+    assert_almost_equal(rtop, gt_rtop, 6)
```

Comment:
It can be some numerical instability of Scipy. I will try to relax a bit the condition, RTOP has very high value ~10^6  so even if it is wrong by 10^(-2) it will be a very good estimation...